src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/UseGenericEventHandlerInstances.cs

```diff
@@ -95,39 +94,14 @@ public override void Initialize(AnalysisContext analysisContext)
                     static bool IsDelegateTypeWithInvokeMethod(INamedTypeSymbol namedType) =>
                         namedType.TypeKind == TypeKind.Delegate && namedType.DelegateInvokeMethod != null;
 
-                    bool IsEventArgsParameter(IParameterSymbol parameter)
-                    {
-                        var type = parameter.Type;
-                        if (IsAssignableTo(type, eventArgs, context.Compilation))
-                        {
-                            return true;
-                        }
-
-                        // FxCop compat: Struct with name ending with "EventArgs" are allowed.
-                        if (type.IsValueType)
-                        {
-                            return type.Name.EndsWith("EventArgs", StringComparison.Ordinal);
-                        }
-
-                        return false;
-                    }
-
-                    bool IsValidNonGenericEventHandler(IMethodSymbol delegateInvokeMethod)
-                    {
-                        return delegateInvokeMethod.ReturnsVoid &&
```

Comment:
A couple of helpers were checking for the `void` return type, I don't know whether that's best to include this in the helper or not.